admin/app/views/spree/admin/orders/_header.html.erb

```diff
@@ -5,33 +5,69 @@
     <%= page_header_back_button spree.edit_admin_order_path(@order) %>
   <% end %>
   <%= @order.number %>
-  <%= order_payment_state(@order, class: 'ml-3') %>
-  <%= order_shipment_state(@order, class: 'ml-3') %>
-  <%= local_time(@order.completed_at, class: 'badge badge-light with-tip ml-3') if @order.completed? %>
+  <%= order_payment_state(@order, class: "ml-3") %>
+  <%= order_shipment_state(@order, class: "ml-3") %>
+  <%= if @order.completed?
+    local_time(@order.completed_at, class: "badge badge-light with-tip ml-3")
+  end %>
+<% end %>
+
+<%# TODO: Fix mobile %>
+<% content_for :page_actions do %>
+
 <% end %>
 
 <% content_for(:page_actions_dropdown) do %>
   <% if @order.completed? && spree.respond_to?(:order_path) %>
-    <%= link_to_with_icon 'eye',
-                          Spree.t(:preview),
-                          spree.order_path(@order, host: current_store.url, token: @order.token),
-                          class: 'text-left dropdown-item',
-                          target: '_blank' %>
+    <%= link_to_with_icon "eye",
+    Spree.t(:preview),
+    spree.order_path(@order, host: current_store.url, token: @order.token),
+    class: "text-left dropdown-item",
+    target: "_blank" %>
   <% end %>
   <% if @order.completed? && can?(:resend, @order) %>
-    <%= link_to_with_icon 'send',
-                          Spree.t(:resend),
-                          resend_admin_order_path(@order),
-                          data: { turbo_method: :post, turbo_confirm: Spree.t(:are_you_sure) },
-                          class: 'btn btn text-left rounded-0 dropdown-item' %>
+    <%= link_to_with_icon "send",
+    Spree.t(:resend),
+    resend_admin_order_path(@order),
+    data: {
+      turbo_method: :post,
+      turbo_confirm: Spree.t(:are_you_sure),
+    },
+    class: "btn text-left dropdown-item" %>
+  <% end %>
+
+  <% if @order.shipments.any?(&:shipped?) %>
+    <% can_create_ra = can?(:create, Spree::ReturnAuthorization) %>
+    <% can_create_cr =
+      can?(:create, Spree::CustomerReturn) && @order.return_authorizations.exists? %>
+
+    <% if can_create_ra || can_create_cr %>
+      <div class="dropdown-divider"></div>
+
+      <% if can_create_ra %>
+        <%= button_link_to Spree.t(:new_return_authorization),
+        new_admin_order_return_authorization_url(@order),
```

Comment:
```suggestion
        spree.new_admin_order_return_authorization_path(@order),
```

1. don't use `_url` helpers, use `_path`
2. always use `spree.` to use spree engines routes